browser/src/map/handler/Map.WOPI.js

```diff
@@ -185,14 +185,14 @@ L.Map.WOPI = L.Handler.extend({
 			return;
 		}
 
+		if (this.DisableInsertLocalImage) {
+			JSDialog.MenuDefinitions.set('InsertImageMenu', []);
```

Comment:
we can't simply set `menuEntriesImage` or `menuEntriesMultimedia` to `[]` like it was prior to https://github.com/CollaboraOnline/online/commit/af317e4130583e29038fd9f45f3a750b11b0cbe5 because this just change the binding not the array itself. That was why some of this wasn't working.